processing/src/test/java/org/apache/druid/segment/virtual/ExpressionVirtualColumnTest.java

```diff
@@ -100,6 +100,12 @@
       ValueType.STRING,
       TestExprMacroTable.INSTANCE
   );
+  private static final ExpressionVirtualColumn ZCONCATNONEXISTENT = new ExpressionVirtualColumn(
```

Comment:
Please add underscores to the name

Reply:
Added, and added them to other statics in the same class.